api/views/submission.py

```diff
@@ -251,8 +255,9 @@ def unclaim(self, request: Request, pk: int, username: str = None) -> Response:
             400: "The volunteer username is not provided",
             403: "The volunteer has not accepted the Code of Conduct",
             404: "The specified volunteer or submission is not found",
-            409: "The submission is already claimed",
+            409: "The submission is already claimed ",
```

Comment:
Do we need a space here?

Reply:
Good catch, this was a remnant from a previous change. Oops!